CHANGELOG.md

```diff
@@ -155,7 +155,14 @@ MobX 5 is the first MobX version fully leveraging Proxies. This has two big adva
 
 ### The system requirements to run MobX has been upped
 
-* MobX 5 can only be used on environments that support `Proxies`. In practice this means, no Internet Explorer (Edge is fine). No nodejs < 6. React Native on Android only when JavaScript core is [upgraded](https://github.com/react-community/jsc-android-buildscripts#how-to-use-it-with-my-react-native-app). All modern browsers are supported.
+* MobX 5 can only be used on environments that support `Proxies`. In practice this means:
```

Comment:
Instead of duplicating this in changelog, it might be better to just link to README.

Reply:
Updated